### ISSUE TITLE
Filter for Logs

### ISSUE THREAD

**Issue by viktor-franz** [FriendsOfShopware/FroshTools#286]
### Please describe the feature you would like to see implemented.

We have installed the FroshTools and we love it. Would be awesome to have a Filter in the Logs to be able to filter the problems faster...

![Image](https://github.com/user-attachments/assets/d2b73d0c-849b-4d7b-82f3-2b8391021f9a)

Thank you!
Would be awesome!

**Comment by viktor-franz** [comment]
Hello, is there a possibility that this filter will be integrated soon?
And a search button for logs of more days would be awesome too :) So that you can search if an error is happening more often! Thank you!

**Comment by shyim** [comment]
I don't think that will be implemented in near future. It's annoying to implement right now as it parses only few lines and filtering and searching would require to load the complete file

**Comment by schneider-felix** [comment]
@viktor-franz I agree with Shyim. We do not want to go too much into the depths of observability with this plugin. If you need more precise analysis capabilities for your logs, I'd recommend using something like Graylog which is purpose build for requirements like these. 

FroshTools on the other hand is meant to improve some day to day tasks of operating a Shopware 6 shops without adding too much overhead. 

As Shyim said, implementing such a feature is really painful as it would involve either reading the entire file into memory or indexing the file into a tool like elasticsearch. 

For now, I will close this issue. If you have other suggestions or issues, feel free to open another issue/pr.
